Pipfile

```
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+PyYAML = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.10"
```

可以降低python版本，别要求太高，对debian友好

```suggestion
python_version = "3"
```

---

为什么不用docker运行呢。用了新版特性，降了跑不起来

---

python的新版特性吗？

---

直接pip install + Pipenv对依赖管理比较成熟（相对于requirements.txt）
我把他打包是因为想二次开发一个BOT

---

用了Python 3.9的特性，不建议降到更低了，如果系统包管理器里面Python太旧了，可以用docker或者conda之类的工具来管理依赖，而不是用旧版。
打成pip包确实不错，以前我没搞，这个可以完善一下，也算是个正式库了。

---

ok，那我改成python_version = "3.9"，我打算直接用这个里面的client写bot

---

确实可以，不过这个client能不能用我还不清楚，是大半年前的东西了，可以试试

---

```suggestion
python_version = "3.9"
```